roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -1,12 +1,66 @@
 ---
 - name: use ceph-volume to create filestore osds with dedicated journals
   command: "ceph-volume lvm create --filestore --data {{ item.data_vg }}/{{ item.data }} --journal {{ item.journal }}"
+  environment:
```

Comment:
Please see: https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-osd/tasks/ceph_disk_cli_options_facts.yml

Perhaps you can factorize some of the tasks using the same approach.

Reply:
I'm not sure that'd work here because each item in ``lvm_volumes`` could be setup different. Meaning one osd could be just data and the next in the list could be data and db.